eclass/cvs.eclass

```diff
@@ -370,7 +378,7 @@ cvs_fetch() {
 			# Force SSH to use SSH_ASKPASS by creating python wrapper
 
 			export CVS_RSH="${T}/cvs_sshwrapper"
-			cat > "${CVS_RSH}"<<EOF
+			cat > "${CVS_RSH}" <<EOF || die
 #!${EPREFIX}/usr/bin/python
```

Comment:
```suggestion
#!${EPREFIX}/usr/bin/env python
```

Assuming that `/usr/bin/python` will exist is wrong. This is only a user-facing convenience, and it is not guaranteed (see `USE=-native-symlinks`). The Python eclasses set up a local symlinks/wrappers in `PATH` to make this work, but then you can't rely on hardcoded path.